components/menus/OkUserMenu.vue

```diff
@@ -51,28 +51,30 @@
                 </nuxt-link>
             </li>
             <li>
-                <nuxt-link :to="'/'"
-                           class="is-disabled has-no-hover-text-decoration ok-has-background-primary-highlight-hover">
+                <a href="javascript://"
+                           @click="onFollowersClick"
+                           class="has-no-hover-text-decoration ok-has-background-primary-highlight-hover">
                             <span class="icon has-padding-right-10">
                                  <ok-followers-icon
                                          class="ok-svg-icon-primary-invert"></ok-followers-icon>
                             </span>
                     <span class="ok-has-text-primary-invert">
                                 {{$t('components.user_dropdown.followers')}}
                             </span>
-                </nuxt-link>
+                </a>
             </li>
             <li>
-                <nuxt-link :to="'/'"
-                           class="is-disabled has-no-hover-text-decoration ok-has-background-primary-highlight-hover">
+                <a href="javascript://"
```

Comment:
if its not really a link, I think just add the @role="button" attr and `has-cursor-pointer`

Reply:
I did the same thing that was done for the settings modal, for example.